spec/models/draft_version_spec.rb

```diff
@@ -40,4 +40,69 @@ class TestObject < ActiveFedora::Base
     end
   end
 
+  describe '#publish!' do
+    let(:user) { FactoryGirl.create(:user) }
+
+    before do
+      @obj = TuftsImage.create(title: 'My title', displays: ['dl'])
+    end
+
+    after do
+      @obj.delete if @obj
+    end
+
+    context "when a published version does not exist" do
+      it "results in a published copy of the draft existing" do
+        published_pid = PidUtils.to_published(@obj.pid)
+        expect(TuftsImage.exists?(published_pid)).to be_falsey
+
+        @obj.publish!
+
+        expect(TuftsImage.exists?(published_pid)).to be_truthy
+      end
+    end
+
+    context "when a published version already exists" do
+      it "destroys the existing published copy" do
+        published_pid = PidUtils.to_published(@obj.pid)
+
+        @obj.publish!
+
+        expect(TuftsImage).to receive(:destroy).with(published_pid).once { true }
+
+        @obj.publish!
+      end
+
+      it "results in a published copy of the original"
```

Comment:
This is a spec I forgot to write! I'm also used to rspec's older behavior where you could leave an unimplemented test and it would result in a "pending" star. Will flesh this test out more.